semgrep.jsonnet

```diff
@@ -84,6 +84,19 @@ local semgrep_rules = [
       `Hashtbl.find_all`.
     |||,
   },
+  {
+    id: 'mask-all-temp-paths',
+    pattern: 'Testo.mask_temp_paths',
+    fix: 'Testutil.mask_temp_paths',
+    message: |||
+      Semgrep applies `Unix.realpath` to some paths resulting in
```

Comment:
would be good to add this comment also before the mask_physical_path above in Testutil.ml